ollama/runtime/src/main/java/io/quarkiverse/langchain4j/ollama/runtime/config/ChatModelConfig.java

```diff
@@ -47,4 +47,12 @@ public interface ChatModelConfig {
      */
     @WithDefault("40")
     Integer topK();
+
+     /**
+     * With a static number the result is always the same. With a random number the result varies
+     * Example:
+     * Random random = new Random();
+     * int x = random.nextInt(Integer.MAX_VALUE);
+     */
+    Integer seed();
```

Comment:
Actually, this needs to be `Optional<Integer>` othewire the property will be required